### DISCUSSION TITLE
Let me show you my solution inspired byt this awesome ideas!

### DISCUSSION THREAD

**Discussion by tomasmark79** [TheLartians/Greeter#196] [Ideas]
Hello,
my name is Tomáš Mark and I am glad that I can share here my project  insipred partialy by this awesome ModernCppStarter.

Let's deep dive into this my project I spent lot of hours to finalize to this useful phase.

https://github.com/tomasmark79/DotNameCpp

### Modern Configurable Project Template for C and C++ 🎁

This template offers a comprehensive solution for quickly starting development in C and C++. It is designed to be easily customizable and enables the creation of modern applications using proven tools and technologies. The template integrates the following components:

- **Visual Studio Code**: Support for the development environment with preconfigured settings.
- **Conan 2**: Dependency management for seamless integration of external libraries.
- **Modern CMake**: Advanced configuration and build system leveraging modern best practices.
- **CPM.cmake**: Simple dependency management without requiring external package installations.
- **CPM.license**: Automated management of third-party licenses.
- **ModernCppStarter**: Inspired by best practices for initializing modern C++ projects.
- **TemplateRenamer**: Easily rename the template to suit your project.
- **Clang and CMake Formatters**: Automated formatting for code and build scripts.

**Comment by viniciusferrao** [comment]
Removed the project @tomasmark79?

**Comment by tomasmark79** [comment]
Renamed long time ago.
https://github.com/tomasmark79/DotNameCpp
